nitrite.go

```diff
@@ -182,6 +178,50 @@ func createAWSNitroRoot() *x509.CertPool {
 	return pool
 }
 
+// Timestamp extracts attestation timestamp from `data` without verifying
+// the attestation.
+func Timestamp(data []byte) (time.Time, error) {
```

Comment:
We use this function only in testing, but I move it here from our PR to `hf/nitrite`. Will probably remove down the road as we refactor more things.

Reply:
Removed it, since we actually don't need it in testing